.github/workflows/documentation.yml

```
@@ -85,18 +79,12 @@ jobs:
         name: Maven cache
         uses: ./.github/actions/maven-cache
 
-      - id: build-documentation
-        name: Build Keycloak
+      - id: build-test-documentation
+        name: Build and verify Keycloak documentation
         shell: bash
         run: |
           MVN_HTTP_CONFIG="-Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=120"
-          ./mvnw install -DskipTests -am -pl docs/documentation/tests -nsu -B -e -Pdocumentation $MVN_HTTP_CONFIG 
-
-      - id: test-documentation
-        name: Verify Keycloak documentation
-        shell: bash
-        run: |
-          ./mvnw test -Dtest=ExternalLinksTest -am -pl docs/documentation/tests,docs/documentation/dist -nsu -B -e -Pdocumentation
+          ./mvnw install -Dtest=ExternalLinksTest -am -pl docs/documentation/tests -nsu -B -e -Pdocumentation $MVN_HTTP_CONFIG 
```

Should we add `docs/documentation/dist` to running tests similarly as it was before? 
```suggestion
          ./mvnw install -Dtest=ExternalLinksTest -am -pl docs/documentation/tests,docs/documentation/dist -nsu -B -e -Pdocumentation $MVN_HTTP_CONFIG 
```

---

@mhajas - I had a look and gave it another try: To run the tests, it needs only the dependencies `docs/documentation/tests` which are the different rendered HTML outputs, not the `docs/documentation/dist`, therefore this additional project is not needed here. 

The `dist` part is only needed in the regular build which is triggered in the `build` which then also uploads these files as a build result.

---

Thanks for looking into this